test/unit/rules/lint-attribute-indentation-test.js

```diff
@@ -0,0 +1,182 @@
+'use strict';
+
+const generateRuleTests = require('../../helpers/rule-test-harness');
+
+generateRuleTests({
+  name: 'attribute-indentation',
+
+  config: true,
+
+  good: [
+    '{{contact-details' + '\n' +
+    '  firstName=firstName' + '\n' +
+    '  lastName=lastName' + '\n' +
+    '}}',
+    //Non Block form with no params
+    '{{contact-details}}',
+    //Non Block form within 70 characters
+    {
+      config: {
+        'open-invocation-max-len': 70
```

Comment:
Just to be clear, this one would pass even without custom config right?

Reply:
Yes, I'll change the example to have more than the default length(80 chars).